Queueomatic/Server/Endpoints/Participant/Delete/DeleteParticipantEndpoint.cs

```diff
@@ -46,11 +46,10 @@ private async Task<bool> IsUserOwnerOfTheRoom(DeleteParticipantRequest req)
         if (user == null) return false;
 
         var isUserTheOwner =
-            user.Rooms.FirstOrDefault(r =>
-                r.Participators.FirstOrDefault(p =>
-                    p.Id == req.Id) != null);
+            user.Rooms.Any(r => r.Participators.FirstOrDefault(p =>
+                        p.Id == req.Id) != null);
 
-        if (isUserTheOwner == null) return false;
+        if (isUserTheOwner == false) return false;
```

Comment:
This code is a bit redundant no? You are checking if it is false and returning false, does that not mean you can return the value directly?

Reply:
You are totally right.